src/charting/index.ts

```diff
@@ -1,5 +0,0 @@
-export { BarChart } from "./charts/BarChart";
```

Comment:
let s keep this file. As i explained it makes it easier to import charts

Reply:
i think i see what you did. Smart! It s related to your change in the typings path?

Reply:
@farfromrefug This is a different file. The new one is still in charts/index.ts.
But we can keep this one too, if you want.